deploy.sh

```diff
@@ -344,49 +344,31 @@ function precheck {
 # Function to handle sensitve data by masking with asterisk
 # -------------------------------------------------------------------------
 function handle_sensitive_data {           
-    
-  ch=''
-  prompt=''
+  stty -echo    
```

Comment:
Please have the code without trailing whitespaces. Always do a "shellcheck" before committing your code.

Reply:
shellcheck plugin I need to download for notepad++?

Reply:
Any URL's for reference to get it in my local machine?

Reply:
https://www.shellcheck.net/